src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -565,10 +567,11 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Specifies the type of virtual machine volumes on which to perform the encryption operation.
-Allowed values for Windows virtual machines: All, OS, and Data.
-The only allowed value for Linux virtual machines: Data.
-The VolumeType parameter is required when encrypting Linux virtual machines, but can be omitted when encrypting Windows virtual machines.
+Specifies the type of virtual machine volumes on which to perform encryption operation: OS, Data, or All. If this parameter is omitted, the operation defaults to All.
```

Comment:
Should shorten this to not specify a default as the following sentences clarify default for Windows and Linux as has no default.

Reply:
Removed the final sentence.